### PR TITLE
setup tailwind

### PR DESCRIPTION
**Pull request by miyabitanimchi**
I configured Tailwind CSS and also set up the color themes for now. (see client/tailwind.config.js).
I didn't really know how to name the colors, so let me know if you have better ones 😅

### CODE REVIEW

**Review by Dannyboy222 — APPROVED** [review]
Thank you Miyabi!

File: client/tailwind.config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  theme: {
+    extend: {
+      colors: {
+      primaryPink: "#f4eeff", // main bg
+      secondaryPurple: "#a6b1e1", // secondary bg
+      lightPurple: "#dcd6f7", // dropdown menu, tech stack tags
+      darkPurple: "#424874", // headers, top titles etc
+      veryDarkPurple: "#2f2e41", // some titles
+      pinkishGray: "#d0cde1", // icons
+      ourGray: "#c4c4c4", // no img
```
Comment (shiv-chan) [inline client/tailwind.config.js:13]: This is my suggestion for name of colour: 
```js
primary-bg-color: "#f4eeff", // main bg & senpai page bg
secondary-bg-color: "#a6b1e1", // kohai page bg
tertiary-bg-color: "#dcd6f7", // dropdown menu, tech stack tags
primary-text-color: "#424874", // headers, top titles etc
```

I couldn't find `veryDarkPurple` on our mockup on Figma. Maybe that's not the colour that I intended to set.

Also, `pinkishGray` for some icons are not needed for now. As long as it's kind of shaded colour that's fine.
`ourGray`that you might find on the dummy profile picture is not needed too. We wanna use some default pictures for it.

@miyabitanimchi Hope it makes sense😄

**Review by shiv-chan — CHANGES_REQUESTED** [review]
@miyabitanimchi I didn't see if it works on my local machine.  I also leave the suggestion for naming colour.

I assume that we are going to use the default font family `font-sans` since it includes Roboto.

**Review by shiv-chan — APPROVED** [review]
LGTM 👍

**Review by Dannyboy222 — APPROVED** [review]
👏

**Pull request merged**